scripts/eject.js

```diff
@@ -104,6 +104,11 @@ prompt('Are you sure you want to eject? This action is permanent. [y/N]', functi
   });
   delete hostPackage.scripts['eject'];
 
+  // explicitly specify ESLint config path for editor plugins
+  hostPackage.eslintConfig = {
+    extends: "./config/eslint.js",
```

Comment:
Thanks @eanplatter, I didn't notice that! 

However, `path.resolve` produces an absolute path which is probably not what we want here (what if the user moves or renames the project folder?). 

How about changing it to `path.normalize('./') + path.normalize('config/eslint.js')`? The first `./` (`.\\` on windows) is required since Atom doesn't work without it.